src/components/button/styles.ts

```diff
@@ -0,0 +1,16 @@
+import { StyleSheet } from 'react-native';
+import { AppColors } from 'src/common/enums';
+
+const styles = StyleSheet.create({
+  global: {
+    borderRadius: 20,
+  },
+  outlined: {
```

Comment:
Use enum for keys here
```suggestion
  [ButtonType.OUTLINED]: {
```